data/tribes/buildings/productionsites/empire/barracks/init.lua

```diff
@@ -23,16 +23,8 @@ descriptions:new_productionsite_type {
       marble = 1
    },
 
-   animations = {
-      idle = { -- TODO(GunChleoc): make animations
-         pictures = path.list_files(dirname .. "idle_??.png"),
-         hotspot = { 56, 68 },
-      },
-      working = { -- TODO(GunChleoc): make animations
-         pictures = path.list_files(dirname .. "idle_??.png"),
-         hotspot = { 56, 68 },
-      }
-   },
+   animation_directory = dirname,
+   animations = {idle = {hotspot = {51, 71}}},
 
```

Comment:
Maybe we can have some sort animation here? E.g. make the windows 'glowing'? Or open the door? Or both :-D

At least the formerly  TODO should be added :-)

Reply:
I would propose an animation similar to frisians (i.e.  a recruit training with the dummy in the foreground)